client/store-management-links/quick-link/index.js

```diff
@@ -19,7 +19,7 @@ export const QuickLink = ( { icon, title, href, linkType, onClick } ) => {
 			<Link
 				onClick={ onClick }
 				href={ href }
-				linkType={ linkType }
+				linktype={ linkType }
```

Comment:
This should be [`type`](https://github.com/woocommerce/woocommerce-admin/blob/main/packages/components/src/link/index.js#L62), no?

Reply:
Thanks @jeffstieler, nice catch! This was changed in the commit 8050514